lib/manager/actions/projects.js

```diff
@@ -112,7 +112,8 @@ export function fetchProjectWithFeeds (projectId: string, unsecure: ?boolean) {
     // Don't fetch project or feeds if project and feeds are already present
     const state = getState()
     const project = state.projects.all.find(p => p.id === projectId)
-    if (project && project.feedSources && project.feedSourceCount && project.feedSources.length === project.feedSourceCount) {
+    const isSaving = state.status.message && state.status.message.includes('Saving')
```

Comment:
I just realized this doesn't support i18n, so this fix won't work in Polish or German. Maybe we should set another redux variable when saving?

Reply:
Indeed. That would also clarify which action this behavior should apply.